src/views/Pipeline.vue

```diff
@@ -423,26 +423,20 @@ export default defineComponent({
               text: this.$t('Skip'),
               handler: async () => {
                 await this.store.dispatch('job/skipJob', job);
-                await this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewIndex: 0, queryString: this.queryString})
+                await this.getPendingJobs();
               },
             }
           ]
         });
       return alert.present();
     },
-    async getPendingJobs(vSize: any, vIndex: any) {
-      const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
-      const viewIndex = vIndex ? vIndex : 0;
+    async getPendingJobs(viewSize: any = process.env.VUE_APP_VIEW_SIZE, viewIndex: any = 0) {
```

Comment:
We need to make them an optional parameter.

Reply:
ok sir